csu

```diff
@@ -127,7 +127,7 @@ defhelp -dev flush 'Run Django flush command.'
 # Run Django makemigrations command
 dev_makemigrations() {
   echo "Creating database migrations..."
-  docker-compose exec django /docker_venv/bin/python3 ./manage.py makemigrations
+  docker-compose exec django /docker_venv/bin/python3 ./manage.py makemigrations --no-input
```

Comment:
Why is this change needed?

Reply:
If you delete a field and add a similar field, django asks if you are renaming the column rather than deleting and adding. When this occurs due to deleting/adding a language, that's a manual Y/N answer for every field for every model. This prevents that user prompt.